src/components/backend-ai-console.ts

```diff
@@ -493,6 +500,10 @@ export default class BackendAIConsole extends connect(store)(LitElement) {
         this.menuTitle = _text("console.menu.Logs");
         this.updateTitleColor('var(--paper-deep-orange-800)', '#efefef');
         break;
+      case 'verify-email':
```

Comment:
Let's use summary and verify-email block together.

Reply:
Merged `verify-email` block with `summary`.